user/tests/integration/slo/connect_time/connect_time.cpp

```diff
@@ -6,13 +6,13 @@ namespace {
 // Number of connection time measurements to make. When changing this parameter, make sure to change
 // the number of cloud_connect_time_from_cold_boot_XX and cloud_connect_time_from_warm_boot_XX tests
 // and update the spec file accordingly
-const unsigned CONNECT_COUNT = 10;
+const unsigned CONNECT_COUNT = 8;
 
 // Delay before resetting the device after a test
 const system_tick_t DELAY_AFTER_TEST = 2000;
 
-// Default timeout for waitFor()
-const system_tick_t WAIT_FOR_TIMEOUT = 10 * 60 * 1000;
+// Test timeout (we have to complete the entire test in 10 minutes, so set this just shy of that with a little overhead)
+const system_tick_t TEST_MAX_TIMEOUT = 9 * 60 * 1000;
```

Comment:
Note that in `.spec.js` you can increase overall timeout driven by test runner with `timeout(xxxx)`.

Reply:
Thanks, I'll keep that in mind for other tests.  I think for this test though, 9 minutes should be long enough to power on/off/connect.  Especially when the test fails if it's more than 30s or 60s to connect.  That would mean we are waiting about 8 minutes to power on/off which is more than enough.